### PR TITLE
Add error handling for cancelled requests

### PR DESCRIPTION
**Pull request by rejas**
Send a statuscode when an axios called is cancelled (e.g. unmount/mount happens when the call is on the way) and pass it upwards.

Also cleanups in the package.json and a proper License

### CODE REVIEW

**Review by iamuchejude — APPROVED** [review]
This is about handling cancelled requests, not about cancelling a request. Got it!
Look good to 🚀

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "nebenan-redux-tools",
   "description": "A set of redux helpers",
   "author": "Good Hood GmbH <tech@nebenan.de>",
-  "license": "SEE LICENSE IN LICENSE FILE",
+  "license": "MIT",
```
Comment (iamuchejude) [inline package.json]: I think we need to talk to someone in legal about this.

Comment (fweinrich) [reply package.json]: i second this, we should clarify this for all public repos

Comment (rejas) [reply package.json]: talked with lars about it, he's ok with it

**Review by fweinrich — APPROVED** [review]

**Comment by rejas** [comment]
Closing since we handle sentry errors in core

**Pull request closed without merging**